static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -0,0 +1,127 @@
+.menu-item {
+    position: relative;
+    width: 100%;
+
+    > *[data-link-name='nav2 : the guardian app'],
+    > *[data-link-name='nav2 : facebook'] {
+        border-top: 1px solid darken($guardian-brand-dark, 4%);
+        margin-top: $gs-baseline / 1.5;
+        padding-top: $gs-baseline / 1.4;
+    }
+
+    > *[data-link-name='nav2 : the guardian app'] {
+        margin-top: $gs-baseline * 2;
+    }
+
+    // First level overwrites
+    .menu-group--primary & {
+        > *:not([open]) {
+            border-bottom: 1px solid darken($guardian-brand-dark, 4%);
+        }
+
+        &:last-child > * {
+            border-bottom: 0;
+        }
+    }
+}
+
+.menu-item__title {
+    color: currentColor;
+    cursor: pointer;
+    display: block;
+    font-size: 18px;
+    outline: none;
+    padding-bottom: $gs-baseline / 1.8;
+    padding-right: $gs-gutter;
+    padding-top: $gs-baseline / 1.8;
+    user-select: none;
+
+    &:hover,
+    &:focus {
+        text-decoration: none;
+    }
+
+    // only for Chrome/ Safari - FF loses the marker via display: block
+    &::-webkit-details-marker {
+        display: none;
+    }
+
+    .menu-group--primary & {
+        font-size: 20px;
+        padding-bottom: $gs-baseline * 1.2;
+        padding-top: $gs-baseline / 2.5;
+
+        @include mq(mobileMedium) {
+            font-size: 22px;
+            padding-top: $gs-baseline / 2.3;
```

Comment:
Again, not so sure about all these `.x` division values. Chat to @zeftilldeath to see if we can better align them with our grid system or else just use a `px` value so at least its less confusing to read.